app/scripts/controllers/journalUpload.js

```diff
@@ -178,7 +177,7 @@ angular
     };
 
     $scope.getRemainingBalance = registrationId => {
-      return _.find($scope.registrations.registrations, {
+      return _.find($scope.registrations, {
```

Comment:
So before ```$scope.registrations``` was the whole response from the api, but now it's specifically just the registration data we care about.